Week_04/Lecture_8/assignment2/src/main/java/org/assignment1/assignment2/repository/employeeRepository.java

```diff
@@ -0,0 +1,41 @@
+package org.assignment1.assignment2.repository;
+
+import org.assignment1.assignment2.model.employee;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public class employeeRepository {
```

Comment:
Please handle exception for these methods.

Reply:
I updated the EmployeeRepository to add every method a handle exception and the example of the result and the code is also updated, Thank you Ms for the guide.